browsergym/core/src/browsergym/core/env.py

```diff
@@ -264,6 +265,14 @@ def override_property(task, env, property):
         self.context.expose_binding(
             "browsergym_page_activated", lambda source: self._activate_page_from_js(source["page"])
         )
+
+        self.context.expose_binding(
```

Comment:
There is no need to modify the BrowserEnv class. This can be done within `task.setup(page)`, using
```python
page.context.expose_binding()
```
This one could also be useful, so that your JS code is executed on all pages visited by the agent (not just the page available at the time of setup())
```python
page.context.add_init_script()
```